.github/workflows/pull_request.yaml

```diff
@@ -106,8 +106,63 @@ jobs:
       working-directory: ./tools/wasme/cli
       run: |
         export PATH=$(dirname $KUBECTL):$PATH
+        export FILTER_IMAGE_GLOO_TAG=webassemblyhub.io/sodman/example-filter:v0.3
         export FILTER_IMAGE_ISTIO_TAG=webassemblyhub.io/ilackarms/assemblyscript-test:istio-1.5
+        export FILTER_BUILD_IMAGE_TAG=localhost:5000/test:v1
+        make install-deps run-tests
+    - name: Debug Info
+      if: failure()
+      env:
+        KUBECTL: ${{ steps.kubectl.outputs.kubectl-path }}
+      run: |
+        # see what's in the cluster if we failed
+        git diff
+        kubectl get all -A
+        kubectl get configmaps -A
```

Comment:
nit: we should probably move this into a script for long term maintenance

Reply:
or a make target

Reply:
just a thought, i realize we have this elsewhere (and i'm not saying now is the ideal time to refactor)

Reply:
I agree, good call. Also in general I think we can do a bunch of cleanup on this GH Actions file to streamline it a bit. I'm thinking we can use the matrix approach we use elsewhere. I tried a few performance optimizations to get the build time down in this PR (as you might see from the commits) but most of them ended up maxing out the CPU of the kind cluster in the e2e test machines and ultimately being rolled back.